.changeset/tasty-cameras-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@apollo/server': patch
+---
+
+Remove Discord link from README.md and templates
```

Comment:
FYI we usually don't bother with this for a doc-only change (and we definitely don't need to mention changes to things like issue templates that don't affect the actual publish package). I guess having some sort of changeset does trigger the new published version which is the only way to get the README change to npmjs.com though. I'll clean this up as part of publishing a version with a different change.

Reply:
Got it, and thank you!